scripts/generate_bindings.sh

```diff
@@ -1,45 +1,49 @@
-#!/usr/bin/env zsh
+#!/bin/sh
 set -e
 set -x
 
 crankstart_crate_dir="$(cd "$(dirname "$0")/.." >/dev/null 2>&1 && pwd)"
-source "$crankstart_crate_dir/scripts/vars.sh" || exit $?
+. "$crankstart_crate_dir/scripts/vars.sh"
+# shellcheck disable=SC2181  # can't check exit code of . in same line with POSIX sh
+if [ "$?" -ne 0 ]; then
+   exit 2
+fi
 
-common_params=("$crankstart_crate_dir/crankstart-sys/wrapper.h"
-  "--use-core"
-  "--ctypes-prefix"         "ctypes"
-  "--with-derive-default"
-  "--with-derive-eq"
-  "--default-enum-style"    "rust"
-  "--allowlist-type"        "PlaydateAPI"
-  "--allowlist-type"        "PDSystemEvent"
-  "--allowlist-type"        "LCDSolidColor"
-  "--allowlist-type"        "LCDColor"
-  "--allowlist-type"        "LCDPattern"
-  "--allowlist-type"        "PDEventHandler"
-  "--allowlist-var"         "LCD_COLUMNS"
-  "--allowlist-var"         "LCD_ROWS"
-  "--allowlist-var"         "LCD_ROWSIZE"
-  "--allowlist-var"         "SEEK_SET"
-  "--allowlist-var"         "SEEK_CUR"
-  "--allowlist-var"         "SEEK_END"
-  "--bitfield-enum"         "FileOptions"
+# POSIX sh "array" used to store common parameters to all bindgen calls
+set -- "$crankstart_crate_dir/crankstart-sys/wrapper.h" \
+  "--use-core" \
+  "--ctypes-prefix"         "ctypes" \
+  "--with-derive-default" \
+  "--with-derive-eq" \
+  "--default-enum-style"    "rust" \
+  "--allowlist-type"        "PlaydateAPI" \
+  "--allowlist-type"        "PDSystemEvent" \
+  "--allowlist-type"        "LCDSolidColor" \
+  "--allowlist-type"        "LCDColor" \
+  "--allowlist-type"        "LCDPattern" \
+  "--allowlist-type"        "PDEventHandler" \
+  "--allowlist-var"         "LCD_COLUMNS" \
+  "--allowlist-var"         "LCD_ROWS" \
+  "--allowlist-var"         "LCD_ROWSIZE" \
+  "--allowlist-var"         "SEEK_SET" \
+  "--allowlist-var"         "SEEK_CUR" \
+  "--allowlist-var"         "SEEK_END" \
+  "--bitfield-enum"         "FileOptions" \
   "--bitfield-enum"         "PDButtons"
-)
 
-bindgen $common_params \
+bindgen "$@" \
   -- \
   -target x86_64 \
   -I"$PLAYDATE_C_API" \
   -DTARGET_EXTENSION > $crankstart_crate_dir/crankstart-sys/src/bindings_x86.rs
```

Comment:
```suggestion
  -DTARGET_EXTENSION > "$crankstart_crate_dir/crankstart-sys/src/bindings_x86.rs"
```

Actually, one quick nit while you're in this file making changes. Can these be quoted, just in case there's a space in the path or something for where you have the crankstart dir?

There are two more occurences below this one too